tox.ini

```diff
@@ -3,26 +3,28 @@ linting_folders=src/wagtailbakery/ tests/ examples/
 
 [tox]
 envlist=
-  py38-django{22,30,31}-wagtail211,                # Wagtail LTS - latest Python - all Django
-  py39-django{22,30,31,32}-wagtail213,             # Wagtail current - latest Python - all Django
-  py{36,37,38}-django32-wagtail213,                # Wagtail current - all Python  - latest Django
+  py310-django{32}-wagtail215,                     # Wagtail 2.15 LTS - latest supported Python - Django LTS
+  py{38,39,310}-django41-wagtail40,                # Wagtail 4.0 - all Python - latest Django
+  py310-django{32,40,41}-wagtail41,                # Wagtail 4.1 LTS - latest Python - all Django
   wagtailmain                                      # Wagtail main latest compatible version
 
 [testenv]
 commands=py.test --cov=wagtailbakery --cov-report=xml {posargs}
 deps=
-  django22: django>=2.2,<2.3       # WT 2.11, 2.13
-  django30: django>=3.0,<3.1       # WT 2.11, 2.13
-  django31: django>=3.1,<3.2       # WT 2.11, 2.13
-  django32: django>=3.2,<3.3       # WT 2.13
-  wagtail211: wagtail>=2.11,<2.12  # Current LTS
-  wagtail213: wagtail>=2.13,<2.14  # Current latest stable
+  django32: django>=3.2,<3.3       # WT 2.15 LTS
+  django40: django>=4.0,<4.1       # WT 4.0, WT 4.1 LTS
+  django41: django>=4.1,<4.2       # WT 4.0, WT 4.1 LTS
+  wagtail215: wagtail>=2.15,<2.16  # Old LTS
+  wagtail40: wagtail>=4.0,<4.1  # Current latest
+
+  ; TODO: use final release when available
+  wagtail41: wagtail==4.1rc1  # Upcoming LTS
```

Comment:
```suggestion
  wagtail41: wagtail>=4.1rc1,<4.2  # Upcoming LTS
```

should do the trick

Reply:
That's pretty cool, did not know this was possible. Thanks for the suggestion Dan ❤️

I will get rid of the TODO comment above as well.